src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--date--1 calendar__day--start-mon"></div>
```

Comment:
your `calendar__day--date--1` just change the first day number
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/102411130/225844414-d9e61abf-527b-43b7-b2b8-905732c8534e.png">

but it should change number of days in a month
<img width="774" alt="image" src="https://user-images.githubusercontent.com/102411130/225844670-034ecbe4-e937-4366-b466-b154b55d1256.png">
<img width="623" alt="image" src="https://user-images.githubusercontent.com/102411130/225844747-e68dc68d-046d-40da-8095-3368bb1becf3.png">